packages/backend.ai-ui/src/components/BAICard.tsx

```diff
@@ -79,10 +80,12 @@ const BAICard: React.FC<BAICardProps> = ({
       )}
       {...cardProps}
       title={
-        <Flex justify="between" align="center" wrap="wrap" gap="sm">
-          {cardProps.title}
-          {_extra}
-        </Flex>
+        cardProps.title || extra ? (
+          <Flex justify="between" align="center" wrap="wrap" gap="sm">
+            {cardProps.title}
+            {_extra}
+          </Flex>
+        ) : null
       }
```

Comment:
I have proposed code changes to address the following issues:
1. Ensure vertical spacing is maintained even when the width is reduced.
2. Fix the issue where the extra content is displayed on the left when only the extra is specified without a title.

```suggestion
      title={
        cardProps.title || extra ? (
          <Flex
            justify={cardProps.title ? 'between' : 'end'}
            align="center"
            wrap="wrap"
            gap="sm"
            style={{
              marginBlock: token.marginSM,
            }}
          >
            {cardProps.title}
            {_extra}
          </Flex>
        ) : null
      }
```